pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[project]
+name = "pocketsizefund"
+version = "0.1.0"
+description = "open source quantitative hedge fund 🍊"
+readme = "README.md"
+requires-python = ">=3.10"
+dependencies = []
+
+[tool.uv.workspace]
+members = ["infrastructure"]
```

Comment:
I struggled with the `uv` PyProject stuff so a clean slate is definitely helpful. Does this mean that the Pulumi definitions are going to be included as part of our overall project? Genuinely curious.

Reply:
my understanding of uv workspaces (which are still kinda new and maybe evolving so can double check) is that this is more like it'll make sure that the whole project has consistent dependencies, but in practice (see my comment to your other comment), with flyte there actually won't really be that many dependencies _per se_, so maybe moot

but, it doesn't mean that if we did

```sh
uv build && uv publish
```

that it would be one package, they are still separate(able) modules.